### PR TITLE
Update php-cs-fixer configuration

### LINKED ISSUE #3884: CS: revert CS changes in Configuration.php
**API Platform version(s) affected**: 2.6.0 alpha

**Description**  

Revert CS changes removing indentation: https://github.com/api-platform/core/commit/7d2cfbb3766b4fab4fd5789759b1321385c7a855

This has probably been changed automatically by PHP CS Fixer. The PHP CS Fixer config must be updated accordingly to prevent any regression.

### PR DESCRIPTION
**Pull request by drupol**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #3884
| License       | MIT

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -598,8 +598,7 @@ private function addDefaultsSection(ArrayNodeDefinition $rootNode): void
                 return $normalizedDefaults;
             });
 
-        [$publicProperties, $configurableAttributes] = ApiResource::getConfigMetadata();
-        foreach (array_merge($publicProperties, $configurableAttributes) as $attribute => $_) {
+        foreach (ApiResource::getConfigMetadata()[1] as $attribute => $_) {
```
Comment (alanpoulain) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php:601]: I don't think this change should be here.

Comment (drupol) [reply src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php:601]: I just reverted the changes introduced in https://github.com/api-platform/core/commit/7d2cfbb3766b4fab4fd5789759b1321385c7a855, I haven't checked if something besides the indentation was changed.

Comment (drupol) [reply src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php:601]: But it makes sense. I will do the change.

Comment (drupol) [reply src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: allrighty, fixed.

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Would be nice to solve the PHPStan issues too (they are not related).

**Comment by drupol** [comment]
> Would be nice to solve the PHPStan issues too (they are not related).

I guess this should be fixed in another ticket, this is a bit out of scope of this ticket. WDYT?

**Comment by alanpoulain** [comment]
We could, but it's always better to have a green CI when merging a PR. It's as you want :slightly_smiling_face: The best way is to make a PR to solve the PHPStan issues, merging it, then rebase this one.

**Comment by alanpoulain** [comment]
Thank you @drupol!

**Pull request merged**